win/pkgs/vsbuildtools.ps1

```diff
@@ -0,0 +1,28 @@
+#Requires -RunAsAdministrator
+
+$ErrorActionPreference="Stop"
+
+$vs_where = "${Env:ProgramFiles(x86)}/Microsoft Visual Studio/Installer/vswhere.exe"
+if (Test-Path $vs_where)
+{
+    $vs_home = & $vs_where -latest -products * -property installationPath
+    if ($vs_home)
+    {
+        Write-Host "Found existing VS installtaion under `"$vs_home`". Skip installation."
+        exit 0
+    }
+}
+
+Invoke-WebRequest -Uri https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile "${Env:SCRATCH}/vs_buildtools.exe"
+
+$install_cmd = "`"${Env:SCRATCH}/vs_buildtools.exe`" --quiet --wait --norestart --nocache " `
```

Comment:
Simply `cmd /c` here since it is never reused.

Reply:
Replied in another thread. Resolved.